### PR TITLE
fix: add Sagnik Sahoo

### PR DESCRIPTION
**Pull request by devgossips**

### CODE REVIEW

**Review by emmalearnscode — APPROVED** [review]

File: public/data/devgossips.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "SAGNIK SAHOO",
```
Comment (Panquesito7) [inline public/data/devgossips.json:2]:
```suggestion
  "name": "Sagnik Sahoo",
```

File: public/data/devgossips.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Contributed to Open-Source",
+      "date": "December 2021",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "❤️ Open Source ",
```
Comment (Panquesito7) [inline public/data/devgossips.json:24]:
```suggestion
      "description": "❤️ Open Source "
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/devgossips.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Contributed to Open-Source",
+      "date": "December 2021",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "❤️ Open Source "
```
Comment (Panquesito7) [inline public/data/devgossips.json:24]:
```suggestion
      "description": "❤️ Open Source"
```

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**